src/ballet/txn/fd_txn.h

```diff
@@ -351,6 +353,17 @@ fd_txn_get_address_tables( fd_txn_t * txn ) {
   return (fd_txn_acct_addr_lut_t *)(txn->instr + txn->instr_cnt);
 }
 
+/* fd_txn_get_signatures: Returns the array of Ed25519 signatures in
+   `payload`, the serialization of the transaction described by `txn`.
+   The number of signatures is seen in `txn->signature_cnt`.
+   U.B. If `payload` and `txn` were not arguments to a valid
+   `fd_txn_parse` call or if either was modified after the parse call. */
+static inline fd_ed25519_sig_t const *
```

Comment:
Can you document that the pointer is within payload so has the same lifetime as payload. Also, it might be badly aligned.